sys/kern/tty.c

```diff
@@ -902,6 +962,54 @@ static int tty_ioctl(file_t *f, u_long cmd, void *data) {
   }
 }
 
+static void tty_hangup(tty_t *tty) {
+  assert(mtx_owned(&tty->t_lock));
+
+  if (!tty_opened(tty) || (tty->t_lflag & CLOCAL))
+    return;
```

Comment:
Could you provide a description why we check CLOCAL flag?

Reply:
Done.